### PR TITLE
[plg_system_sef] Add sef URI when using srcset attribute

### PR DESCRIPTION
**Pull request by andrepereiradasilva**
Pull Request for Issue https://github.com/joomla/joomla-cms/issues/10463.
#### Summary of Changes

See https://github.com/joomla/joomla-cms/issues/10463
#### Testing Instructions
- Use SEF URL (global config + sef system plugin)
- Add an image in an article with this html (or equivalent)

``` html
<img src="images/sampledata/parks/banner_cradle.jpg" srcset="images/sampledata/parks/banner_cradle.jpg" alt="yyy" />
```
- Check in the frontend the srcset attribute URL is NOT converted to SEF URL (misses the base path).
- Apply patch
- Check in the frontend the srcset attribute URL is converted to SEF URL (in this example, should be `/images/sampledata/parks/banner_cradle.jpg` or `/<sitepath>/images/sampledata/parks/banner_cradle.jpg`)

Please note this will not solve all `srcset` possibilities, only for cases like the one described in the issue (path to one image).

### CODE REVIEW

File: plugins/system/sef/sef.php [hunk 1]
```diff
@@ -107,7 +107,7 @@ public function onAfterRender()
 
 		// Check for all unknown protocals (a protocol must contain at least one alpahnumeric character followed by a ":").
 		$protocols = '[a-zA-Z0-9\-]+:';
-		$attributes = array('href=', 'src=', 'poster=');
+		$attributes = array('href=', 'src=', 'srcset=', 'poster=');
 		foreach ($attributes as $attribute)
 		{
```
Comment (zero-24) [inline plugins/system/sef/sef.php]: Can you add a clean line bevor the foreach?

**Comment by andrepereiradasilva** [comment]
@zero-24 this PR is not for code style of the SEF plugin ...

**Comment by wojsmol** [comment]
@zero-24  Please see https://github.com/joomla/joomla-cms/pull/10465

**Comment by brianteeman** [comment]
I have tested this item :white_check_mark: successfully on 84743103b45937f64fe194a035c4961cc83d2a16<br /><br />Issue confirmed and patch fixes it<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10464">issues.joomla.org/joomla-cms/10464</a>.</sub>

**Comment by infograf768** [comment]
I have tested this item :white_check_mark: successfully on 84743103b45937f64fe194a035c4961cc83d2a16<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10464">issues.joomla.org/joomla-cms/10464</a>.</sub>

**Comment by infograf768** [comment]
RTC. Thanks<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10464">issues.joomla.org/joomla-cms/10464</a>.</sub>

**Comment by infograf768** [comment]
for 3.6.0 I guess.

**Comment by roland-d** [comment]
Thanks everybody

**Pull request merged**